web/webkit/src/main/scala/net/liftweb/http/provider/HTTPProvider.scala

```diff
@@ -87,12 +87,29 @@ trait HTTPProvider {
         preBoot
         b.boot
       } catch {
+        // The UnavailableException is the idiomatic way to tell a Java application container that
+        // the boot process has gone horribly, horribly wrong. That _must_ bubble to the application
+        // container that is invoking the app. See https://github.com/lift/framework/issues/1843
+        case unavailableException: javax.servlet.UnavailableException =>
+          logger.error("------------------------------------------------------------------")
+          logger.error("------------------------------------------------------------------")
+          logger.error("------------------------------------------------------------------")
+          logger.error("------------------------------------------------------------------")
```

Comment:
I'm not super opinionated on this, personally, but I'm inclined to leave it as is for now and reconsider when we next touch this code.